res/controllers/Roland_DJ-202-scripts.js

```diff
@@ -107,6 +107,54 @@ DJ202.Deck = function (deckNumbers, offset) {
 
     this.paramPlusMinus = new components.Button({
         deck: this,
+        songKeyMode: false,
+        setLEDs: function (plusValue, minusValue) {
```

Comment:
Niiice! I missed some way to reset the key, but this is even better.

Reply:
Currently, holding shift+param⁻+param⁺ at the same time will reset the key. I’ve found it a bit fumbly to actuate though. We might want to change that key-reset mapping to be bound to keylock double-tap.